### PR TITLE
AvalancheGo 1.9.5 Release Notes

### PR DESCRIPTION
**Pull request by rob-ava**
Changes

### CODE REVIEW

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 1]
```diff
@@ -6,6 +6,18 @@
 
 :::
 
+## V1.9.5 ([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.6))
+
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:10]: We should add a description for this release as in 
Screen Shot 2022-12-19 at 9 12 55 AM

We should add it to 1.9.4 too.

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 2]
```diff
@@ -6,6 +6,18 @@
 
 :::
 
+## V1.9.5 ([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.6))
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:9]: It should be 1.9.5 in the link to github.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 3]
```diff
@@ -6,6 +6,18 @@
...
+
+This version is backwards compatible to
+[v1.9.0](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0). It is
+optional, but encouraged. The supported plugin version is `20`.**
+
+**Change 1**
```
Comment (cgcardona) [inline docs/apis/avalanchego/avalanchego-release-notes.md:15]: I added a *DO NOT MERGE* label until these change details are fleshed out.

Comment (joshua-kim) [reply docs/apis/avalanchego/avalanchego-release-notes.md:15]: We might want to make these PR drafts in the future so we can still leave feedback on them, but block merging automatically

File: docs/apis/avalanchego/avalanchego-release-notes.md [hunk 4]
```diff
@@ -6,16 +6,43 @@
...
 
-## V1.9.5 ([View on GitHub](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.6))
+## [v1.9.5](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.5)
 
-This version is backwards compatible to
-[v1.9.0](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0). It is
-optional, but encouraged. The supported plugin version is `20`.**
+This version is backwards compatible to [v1.9.0](https://github.com/ava-labs/avalanchego/releases/tag/v1.9.0). It is optional, but encouraged. The supported plugin version is `21`.
+
+### Subnet Messaging
```
Comment (yulin-dong) [inline docs/apis/avalanchego/avalanchego-release-notes.md:13]: All of these should be changed to ** **

**Pull request closed without merging**